### DISCUSSION TITLE
Como aprender e entrar no mercado de UX

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#137] [Perguntas e Respostas]
Fala galera, sei que aqui é um fórum de frontend, mas sempre tive duvidas sobre como aprender e entrar no mercado de UX, é algo relativamente novo e ainda meio distorcido de como aprender e oq é necessário saber para entrar no mercado, alguém aqui tem experiência sobre o assunto?

**Comment by woliveiras** [comment]
Eita, que esse tópico seria da hora abrir [aqui](https://github.com/training-center/mentoria) também ein mano.

**Comment by felipefialho** [comment]
Não se preocupa @Cleysonlb, acredito que Front-End e UX estão totalmente ligadas, inclusive acabei de criar uma label sobre o assunto.

De cara eu recomendo você participar do sensacional Meetup [Front UX](http://www.meetup.com/pt-BR/FrontUX/events/230197224/) do @matmarsiglio e do @gmoura. Você vai ter contato com gente que trabalha diretamente com UX, mas com a visão da galera de Front-End participando dos debates. Imperdível.

Acompanhe blogs sobre o assunto, posso recomendar:
- [Arquitetura De Informação](http://arquiteturadeinformacao.com/)
- [UX.BLOG](http://www.uxdesign.blog.br/)  
- [Babich](http://babich.biz/)
- [UX Planet](https://uxplanet.org/)

Na minha opinião, os melhores atualmente.

Não sei se me segue no twitter, mas costumo postar muito conteúdo de UX.

Por último o @MarcoBruno tem feito uma serie de artigos muito legais sobre UX e se aprofundado muito no tema, recomendo segui-lo no Twitter para acompanhar as novidades: https://twitter.com/marcobrunobr

**Comment by Cleysonlb** [comment]
Boa @LFeh te sigo sim la no twitter, cara realmente esses sites tem muito conteúdo, tem até podcast \o
Vou estudar mais sobre o assunto, sempre me identifiquei mais com a parte de UI, UX mesmo no frontend.

**Comment by rickbenetti** [comment]
@Cleysonlb eu to estudando e migrando de área, deixando de lado o FrontEnd e voltando pro Design e nisto inclui-se o UX/UI Design, e com isso criei uma lista no meu Twitter [UX/UI Design](https://twitter.com/_RKB81/lists/ux-ui-design) e tem um canal meu no medium [WDSIGN](https://wdsign.com.br) onde estou compartilhando o que estou aprendendo, um pouco devagar, mas tá indo.

**Comment by Cleysonlb** [comment]
Opa, vlw por compartilhar, vou acompanhar seu medium @rkb81 👍

**Comment by jonathanslima** [comment]
Um podcast brazuca sobre UX @Cleysonlb https://soundcloud.com/movimento-ux

**Comment by rubenmarcus** [comment]
@rkb81  o logo do seu site era pra ser um circulo? aqui esta cortado no final

e essa fonte parece não estar bem renderizada...

**Comment by Cleysonlb** [comment]
Cara muito legal o conteúdo dos blogs e podcasts, mas a duvida persiste, exemplo, se quero virar front end eu pego e aprendo HTML, CSS, JS e fico fazendo sites, mexendo no código etc, mesma coisa com backend pra aprender e testar, e no UX, oq posso fazer pra começar e quem sabe usar como "portfólio" pra um futuro job na área?

**Comment by Marcelosilva10** [comment]
Galera recomendo para vocês esse cara já ouviu falar @woliveiras , @LFeh 

[https://www.youtube.com/channel/UC0f9NbKjUUbnL2cOa12ENIQ](Mike Locke) ~ Mike Locke UX

**Comment by rickbenetti** [comment]
@rubenmarcus sim acabei subindo um logo velho, é o WDSIGN dentro de um círculo, por lembrar inovação e o globo.

**Comment by rickbenetti** [comment]
@Marcelosilva10 corrigi o link, o texto é um lugar e a URL tá indo pra outro lugar errado.
